.evergreen.yml

```diff
@@ -50,6 +54,31 @@ functions:
         working_dir: spruce/evergreen
         command: make configure-mongod
 
+  setup-node:
+    - command: subprocess.exec
+      params:
+        binary: bash
+        args:
+          - spruce/scripts/prepare-shell.sh
+    - command: expansions.update
+      params:
+        file: expansion.yml
+    - command: shell.exec
+      params:
+        script: |
+          ${PREPARE_SHELL}
+
+          # Fetch NVM and install it into this task's .nvm directory
+          # Once downloaded, source nvm and install yarn
+          git clone https://github.com/nvm-sh/nvm.git "$NVM_DIR"
```

Comment:
Might be worthwhile asking mathew to see if we can prioritize the build NVM ticket. And also open a follow-up ticket that's linked to it to use it.